yarn-project/noir-compiler/src/compile/noir/dependencies/github-dependency-resolver.ts

```diff
@@ -0,0 +1,133 @@
+import { createDebugOnlyLogger } from '@aztec/foundation/log';
+
+import { join, sep } from 'node:path';
+import { unzip } from 'unzipit';
+
+import { FileManager } from '../file-manager/file-manager.js';
+import { NoirDependencyConfig, NoirGitDependencyConfig } from '../package-config.js';
+import { NoirPackage } from '../package.js';
+import { DependencyResolver } from './dependency-resolver.js';
+
+/**
+ * Downloads dependencies from github
+ */
+export class GithubDependencyResolver implements DependencyResolver {
+  #fm: FileManager;
+  #log = createDebugOnlyLogger('');
```

Comment:
```suggestion
  #log = createDebugOnlyLogger('aztec:compile:github-dependency-resolver');
```
So we can turn on debug for aztec by exporting `DEBUG=aztec:*`.